mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule Scrivener.Mixfile do
 
   defp deps do
     [
-      {:ecto, "~> 1.0"},
+      {:ecto, ">= 1.0.0"},
```

Comment:
This is saying scrivener will work with Ecto 1, 2, 3, 4... hence `~> 1.0 or ~> 2.0` that will set it to only Ecto 1 and 2. :)